build/appveyor/MSYS-appveyor-install.bat

```diff
@@ -28,11 +28,11 @@ CALL cl_showenv.bat                        || EXIT /B
 SET PACKAGES=^
   --needed -S bison flex ^
   make ^
-  openssl openssl-devel ^
+  mingw-w64-openssl ^
```

Comment:
Thanks for noticing the missing architecture! Fixed in latest PR.
The `mingw-w64` variants of packages do not use dev / devel package.